WMF Framework/ReadingListsSyncOperation.swift

```diff
@@ -819,10 +814,25 @@ internal class ReadingListsSyncOperation: ReadingListsOperation {
                     return
                 }
                 
-                guard let entry = NSEntityDescription.insertNewObject(forEntityName: "ReadingListEntry", into: moc) as? ReadingListEntry else {
+                var entry = ReadingListEntry(context: moc)
```

Comment:
If this is created and the guard below returns, could this end up being saved as an empty (= all attributes would be nil) object? 🤔

Reply:
good catch, fixed